### PR TITLE
task9'

### PR DESCRIPTION
**Pull request by YarasKo**

### CODE REVIEW

File: Twhatter/Twhatter/src/main/webapp/resources/css/style.css [hunk 1]
```diff
@@ -123,7 +123,7 @@ body {
 
 }
 
-.post-container {
+.post-template {
```
Comment (lexastef) [inline Twhatter/Twhatter/src/main/webapp/resources/css/style.css:126]: That will not work as `<template> ` content is not rendered directly.

File: Twhatter/Twhatter/src/main/webapp/resources/js/view.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
+'use strict';
+class view{
```
Comment (lexastef) [inline Twhatter/Twhatter/src/main/webapp/resources/js/view.js:2]: class name from upper case!

File: Twhatter/Twhatter/src/main/webapp/resources/js/view.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    template;
+    container;
+    constructor() {
+        // Caching template
```
Comment (lexastef) [inline Twhatter/Twhatter/src/main/webapp/resources/js/view.js:6]: +0.5

File: Twhatter/Twhatter/src/main/webapp/resources/js/view.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+                phElement.textContent = String(data[key]);
+                if(key==='author'&&data[key]!=currentUser){
+                    let el = item.querySelector('.smallerbutton');
+                    el.style='display: none;';
+                }
+            });
+        }
+        clear(){
+            const parent = document.getElementById("notes");
+            while (parent.firstChild) {
```
Comment (lexastef) [inline Twhatter/Twhatter/src/main/webapp/resources/js/view.js:30]: parent.innerHTML = '';

File: Twhatter/Twhatter/src/main/webapp/resources/js/view.js (hunk 2 above)
Comment (YarasKo) [reply Twhatter/Twhatter/src/main/webapp/resources/js/view.js:2]: When the semester is over, I'll read a book about clean code. I promise

**Pull request merged**